### PR TITLE
CiviForm Logo as "home" navigation to program page

### LINKED ISSUE #3935: CiviForm Logo as "home" navigation to program page
**Is your feature request related to a problem? Please describe.**
As a CiviForm Admin, it would make things more intuitive if the CiviForm Logo in the top left was actually a "home" navigation button. It would be helpful if it navigated back to the Admin Programs home page. Currently, the logo doesn't do anything. And there is no clear navigation to go from reviewing applications back to the Admin Program home page.

**Describe the solution you'd like**
Clicking the CiviForm logo as a CiviForm Admin should take me to the Admin Programs Page

**Done when**
When I'm logged in as a CiviForm Admin and I click the CiviForm logo, I am routed back to the Admins Program Page.

image

### PR DESCRIPTION
**Pull request by swatkat1**
### Description
CiviForm Logo as "home" navigation to program page.

## Release notes

CiviForm Logo as "home" navigation to program page. 

#### General

- [x] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [x] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

#### User visible changes

- [x] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [x] Manually tested at 200% size
- [ ] Manually evaluated tab order

#### New Features

- [ ] Add new FeatureFlag env vars to `server/conf/application.conf`
- [ ] Conditioned new functionality on a [FeatureFlag](https://docs.civiform.us/contributor-guide/developer-guide/feature-flags)
- [ ] Wrote browser tests with the feature flag off and on, etc.

### Issue(s) this completes

Fixes #3935

### CODE REVIEW

File: server/app/views/admin/AdminLayout.java [hunk 1]
```diff
@@ -38,6 +38,7 @@ public enum NavPage {
   private final NavPage activeNavPage;
 
   private AdminType primaryAdminType = AdminType.CIVI_FORM_ADMIN;
+  // private final FakeAdminClient fakeAdminClient;
```
Comment (gwendolyngoetz) [inline server/app/views/admin/AdminLayout.java:41]: Please remove the unused code.

File: server/app/views/admin/AdminLayout.java [hunk 2]
```diff
@@ -84,8 +85,11 @@ private NavTag renderNavBar() {
     String logoutLink = org.pac4j.play.routes.LogoutController.logout().url();
 
     DivTag headerIcon =
-        div(span("C"), span("F").withClasses("font-thin"))
-            .withClasses(AdminStyles.ADMIN_NAV_BAR_LOGO);
+        div(
+            a().withHref(controllers.routes.HomeController.index().url())
```
Comment (gwendolyngoetz) [inline server/app/views/admin/AdminLayout.java:89]: Can you double check on the issue? I would think you want to redirect to `AdminProgramController` and not `HomeController`.

**Review by gwendolyngoetz — CHANGES_REQUESTED** [review]

File: server/app/views/admin/AdminLayout.java (hunk 1 above)
Comment (swatkat1) [reply server/app/views/admin/AdminLayout.java:41]: Done!

File: server/app/views/admin/AdminLayout.java (hunk 2 above)
Comment (swatkat1) [reply server/app/views/admin/AdminLayout.java:89]: Hi Gwen, Calling the AdminProgramController always returns a 403 error. It is the homecontroller which calls the other controllers based on the profile- https://github.com/civiform/civiform/blob/95caf4c71e2a2488039e2b01342609bd781ee10b/server/app/controllers/HomeController.java#L51
public CompletionStage<Result> index(Http.Request request) {
    Optional<CiviFormProfile> maybeProfile = profileUtils.currentUserProfile(request);

    if (maybeProfile.isEmpty()) {
      return CompletableFuture.completedFuture(
          redirect(controllers.routes.HomeController.loginForm(Optional.empty())));
    }

    CiviFormProfile profile = maybeProfile.get();

    if (profile.isCiviFormAdmin()) {
      return CompletableFuture.completedFuture(
          redirect(controllers.admin.routes.AdminProgramController.index()));
    } else if (profile.isProgramAdmin()) {
      return CompletableFuture.completedFuture(
          redirect(controllers.admin.routes.ProgramAdminController.index()));
    } else {

**Review by gwendolyngoetz — APPROVED** [review]

**Pull request merged**